### DISCUSSION TITLE
Support SAML2 authentication.

### DISCUSSION THREAD

**Discussion by benanhalt** [specify/specify7#5525] [Feature Requests]

**Comment by maxpatiiuk** [comment]
@benanhalt is this fixed?

**Comment by benanhalt** [comment]
There is a branch that implements SAML2 auth but it has not been tested.

**Comment by grantfitzsimmons** [comment]
Not fixed.

**Comment by grantfitzsimmons** [comment]
Should be fixed by https://github.com/specify/specify7/pull/4154 if we finish it

**Requested by:** UBC
